frontend/src/app/modules/theme/theme-actions.js

```diff
@@ -1,7 +1,7 @@
 import { createActions } from 'reduxsauce'
 
 export const ThemeActions = createActions({
-  themeChanged: ['newTheme'],
+  themeChanged: ['theme'],
   changeTheme: ['theme'],
```

Comment:
Maybe change the argument name to 'currentTheme' or something similar - just by seeing this, I'd think that the 'theme' argument holds the 'theme' you want to change to.

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/81/commits/44f0448e166dac047262d9a907d4cc973e9a7619#diff-ebe66918bbf9967140fef8b0899e13f1)